ownCloud/Static Login/StaticLoginProfile.swift

```diff
@@ -41,60 +42,270 @@ class StaticLoginProfile: NSObject {
+					.category	: "Branding",
+					.subCategory	: "Profile"
+				],
+
+				StaticLoginProfile.Key.welcome.settingsKey : [
+					.type 		: OCClassSettingsMetadataType.string,
+					.label 		: "Welcome Message",
+					.description	: "Welcome message shown during account setup.",
+					.status		: OCClassSettingsKeyStatus.advanced,
+					.category	: "Branding",
+					.subCategory	: "Profile"
+				],
+
+				StaticLoginProfile.Key.bookmarkName.settingsKey : [
+					.type 		: OCClassSettingsMetadataType.string,
+					.label		: "Bookmark Name",
+					.description	: "The name that should be used for the bookmark that's generated from this profile and appears in the account list.",
+					.status		: OCClassSettingsKeyStatus.advanced,
+					.category	: "Branding",
+					.subCategory	: "Profile"
+				],
+
+				StaticLoginProfile.Key.url.settingsKey : [
+					.type 		: OCClassSettingsMetadataType.urlString,
+					.label		: "URL",
+					.description 	: "The URL of the server targeted by this profile.",
+					.status		: OCClassSettingsKeyStatus.advanced,
+					.category	: "Branding",
+					.subCategory	: "Profile"
+				],
+
+				StaticLoginProfile.Key.helpURL.settingsKey : [
+					.type 		: OCClassSettingsMetadataType.urlString,
+					.label 		: "Help URL",
+					.description	: "Optional URL to help/support resources.",
+					.status		: OCClassSettingsKeyStatus.advanced,
+					.category	: "Branding",
+					.subCategory	: "Profile"
+				],
+
+				StaticLoginProfile.Key.promptForHelpURL.settingsKey : [
+					.type 		: OCClassSettingsMetadataType.string,
+					.label		: "Open Help message",
+					.description 	: "Message shown in an alert before opening the help URL.",
+					.status		: OCClassSettingsKeyStatus.advanced,
+					.category	: "Branding",
+					.subCategory	: "Profile"
+				],
+
+				StaticLoginProfile.Key.helpURLButtonString.settingsKey : [
+					.type 		: OCClassSettingsMetadataType.string,
+					.label		: "Help button label",
+					.description 	: "Text used for the Help button label",
+					.status		: OCClassSettingsKeyStatus.advanced,
+					.category	: "Branding",
+					.subCategory	: "Profile"
+				],
+
+				StaticLoginProfile.Key.canConfigureURL.settingsKey : [
+					.type 		: OCClassSettingsMetadataType.boolean,
+					.label	 	: "Allow URL configuration",
+					.description	: "Indicates if the user can change the server URL for the account.",
+					.status		: OCClassSettingsKeyStatus.advanced,
+					.category	: "Branding",
+					.subCategory	: "Profile"
+				],
+
+				StaticLoginProfile.Key.allowedAuthenticationMethods.settingsKey : [
+					.type 		: OCClassSettingsMetadataType.stringArray,
+					.label		: "Allowed authentication methods",
+					.description 	: "The identifiers of the authentication methods allowed for this profile. Allows to f.ex. force OAuth2, or to use Basic Auth even if OAuth2 is available. See connection.allowed-authentication-methods for possible values.",
+					.status		: OCClassSettingsKeyStatus.advanced,
```

Comment:
Does it makes sense to add the possible values? This will show inside the documentation which values are possible for that field.

```
.possibleValues : [
"Basic Auth" : "com.owncloud.basicauth", 
"OAuth2" : "com.owncloud.oauth2"
]
```

Reply:
Yeah, it absolutely makes sense. I changed the SDK to make the dynamically generated values of `connection.allowed-authentication-methods` easily available for this, then returned from here, so it's now inlined as a table into the documentation.